decidim-conferences/app/views/decidim/conferences/conferences/show.html.erb

```diff
@@ -14,71 +14,155 @@ edit_link(
 )
 %>
 
+<% content_for :aside do %>
+  <div class="conference__nav-container">
+    <ul class="conference__nav">
```

Comment:
The conference navigation has the [same problem on mobile that we had for pages](https://github.com/decidim/decidim/pull/9457#discussion_r943554546):
![image](https://user-images.githubusercontent.com/864340/187424103-18678aae-0cc5-4096-8650-9ee9fc796183.png)

The user has to scroll a lot to get to the actual content of this page.

I'm not sure if this is intentional as finding the sub-navigation has been also a problem for many users but it is also a problem when the navigation takes so much screen estate taking the user's attention.

Reply:
Moved to https://github.com/decidim/decidim/issues/9679